### ISSUE TITLE
error converting YAML to JSON: yaml

### ISSUE THREAD

**Issue by challengelogic** [dotnet/core#5191]
Problem encountered on https://dotnet.microsoft.com/learn/aspnet/deploy-microservice-tutorial/deploy
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

Error on step: **kubectl apply -f deploy-myMicroservice.yaml**

Error output: C:\Users\Darren\myMicroservice>kubectl apply -f deploy-myMicroservice.yaml
deployment.apps/mymicroservice unchanged
**error: error parsing deploy-myMicroservice.yaml: error converting YAML to JSON: yaml: line 12: could not find expected ':'**

**Comment by mairaw** [comment]
@nishanil @shirhatti can you help here?

**Comment by shirhatti** [comment]
@challengelogic Please share the contents of your `deploy-myMicroservice.yaml`?

**Comment by challengelogic** [comment]
Its identical to the one in the documentation step.

I managed to solve the issue by putting the original YAML code into a JSON converter, then updating the JSON side 9f code with my github location ID and then copying the newly created YAML file from the converter.

https://www.json2yaml.com/convert-yaml-to-json

I then pasted the new YAML code into VS Code and ran the command listed in the documentation, which worked successfully.

That was the only way the YAML could be parsed.



Sent from Samsung Mobile on O2



-------- Original message --------
